include/rcutils/logging.h

```diff
@@ -58,6 +58,8 @@ extern bool g_rcutils_logging_initialized;
  * \return `RCUTILS_RET_LOGGING_SEVERITY_MAP_INVALID` if the internal logger
  *   severity map cannot be initialized, in which case logger severity
  *   thresholds will not be configurable.
+ * \return `RCUTILS_RET_MULTIPLE_ERRORS` if multiple errors occur, in which
+ *   case the error message will contain that of the last error to occur.
```

Comment:
Mmm, I don't think this makes sense. Consider a case where an invalid argument error also produces a bad alloc while formatting the error message for instance... It would be better to receive either ret code than this one imo. With this error code you only know that something went wrong, but not what went wrong without parsing the error string (which is slow and might change over time). Part of the point of the ret codes is so the calling code can react to the issue within the called function, but this return code doesn't give you any useful information in that respect. I'd rather have one of the errors printed to stderr, or added to the error message, and the other one returned normally (with error message set and the proper return code).

Reply:
ok, I see now that "return an error return code if there are any accumulated" in https://github.com/ros2/rmw_connext/issues/265 doesn't necessarily mean it is a "special" error code. I follow your reasoning and will change it to be one of the existing codes

Reply:
13cebe7634c0f99f893e497385cae0572b451253 + b9d749b6b4e5bdce38ef9f376026b8d42b54c050

Reply:
I was thinking, for that other comment, that it would return multiple return codes, maybe a list of them or something.

Reply:
ah ok, thanks for clarifying (I can't find the full context of that quote of yours so I'm inventing a lot of context! )